lotus.go

```diff
@@ -332,6 +330,85 @@ func lotusSendFIL(ctx context.Context, fromAddr, toAddr address.Address, filAmou
 	return sm.Cid(), nil
 }
 
+var ErrNotMiner = errors.New("not a miner")
+
+func lotusTranslateError(err *error) {
+	if *err == nil {
+		return
+	}
+	if strings.Contains((*err).Error(), "not found") {
+		*err = ErrNotMiner
+	}
+}
+
+func lotusGetMinerAddr(ctx context.Context, addr address.Address) (_ address.Address, err error) {
+	defer withStack(&err)
+	defer lotusTranslateError(&err)
+
+	api, closer, err := lotusGetFullNodeAPI(ctx)
+	if err != nil {
+		return address.Undef, err
+	}
+	defer closer()
+
+	if addr.Protocol() != address.ID {
+		// if this call errs, we're not gonna check explicit err messages
+		// just assume this is not a miner
+		idAddr, err := api.StateLookupID(ctx, addr, types.EmptyTSK)
+		if err != nil {
+			return address.Undef, ErrNotMiner
```

Comment:
Instead of using this `ErrNotMiner` val, lets actually just send back an error here which is like:

`try passing an ID address` - this way the frontend UI can explicitly tell the user to pass in a t0 address

Reply:
This is never actually sent to the frontend.  The function that calls this function does this:
```go
	// returns an ID address if its a miner address, otherwise an empty address
	minerAddr, err := lotusGetMinerAddr(ctx, targetAddr)
	if err != nil && errors.Cause(err) != ErrNotMiner {
		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
		return
	}

	isMiner := !minerAddr.Empty()

	//...
```
Having a specific error value to check against is safer and more future-proof than comparing error strings, so I'd vote we keep this.  Thoughts?